[depth/color]: Failed to start stream

Hi @alexvannobel ,

When I doing the '#~/catkin_ws/src/ros_kortex_vision/launch$ roslaunch kinova_vision.launch', the terminal will show the following errors... And the image data cannot be obtained in RViz.

<img width="518" alt="kortex_topic" src="https://user-images.githubusercontent.com/89619820/135029550-2e041b5a-23f0-4231-ba79-aafef35ad383.png">

----------
process[camera/camera_nodelet_manager-1]: started with pid [55127]
process[camera/depth/kinova_vision_depth-2]: started with pid [55128]
process[camera/color/kinova_vision_color-3]: started with pid [55129]
process[camera/camera_depth_tf_publisher-4]: started with pid [55130]
process[camera/camera_color_tf_publisher-5]: started with pid [55131]
process[camera/color_rectify_color-6]: started with pid [55145]
process[camera/depth_rectify_depth-7]: started with pid [55153]
process[camera/depth_metric_rect-8]: started with pid [55157]
[ INFO] [1632800861.267624745]: Initializing nodelet with 4 worker threads.
process[camera/depth_metric-9]: started with pid [55165]
process[camera/depth_points-10]: started with pid [55174]
[ INFO] [1632800861.295277407]: Using gstreamer config from rosparam: "rtspsrc location=rtsp://192.168.1.10/depth latency=30 ! rtpgstdepay"
[ INFO] [1632800861.351253797]: Using gstreamer config from rosparam: "rtspsrc location=rtsp://192.168.1.10/color latency=30 ! rtph264depay ! avdec_h264 ! videoconvert"

[ERROR] [1632800861.683547761]: [depth]: Failed to start stream
[ INFO] [1632800861.684516204]: [depth]: Trying to connect... (attempt #1)
[ERROR] [1632800862.087753582]: [color]: Failed to start stream
[ INFO] [1632800862.089763737]: [color]: Trying to connect... (attempt #1)
[ERROR] [1632800865.022285919]: [depth]: Failed to start stream
[ INFO] [1632800865.023103441]: [depth]: Trying to connect... (attempt #2)
[ERROR] [1632800865.424782482]: [color]: Failed to start stream
[ INFO] [1632800865.426127682]: [color]: Trying to connect... (attempt #2)
[ERROR] [1632800868.346317575]: [depth]: Failed to start stream

--- a/include/constants.h
+++ b/include/constants.h
@@ -8,7 +8,7 @@ const std::string DEFAULT_DEPTH_FRAME_ID = "camera_depth_frame";
 const std::string DEFAULT_COLOR_FRAME_ID = "camera_color_frame";
 
 const float RETRY_INTERVAL = 3.0;
-const unsigned int STATE_CHANGE_ASYNC_TIMEOUT = 5;
+const unsigned int STATE_CHANGE_ASYNC_TIMEOUT = 15;
 const unsigned int APP_SINK_BUFFER_COUNT = 5;
 
 #endif

